tests/iot_config.h

```diff
@@ -88,6 +88,7 @@
     #define IOT_MQTT_CONNECTIONS                   ( 2 )
     #define IOT_MQTT_MAX_IN_PROGRESS_OPERATIONS    ( 10 )
     #define IOT_MQTT_SUBSCRIPTIONS                 ( 80 )
+    #define IOT_TASKPOOLS_INSTANCES                ( 10 )
```

Comment:
Minor: Rename to `IOT_TASKPOOLS` for consistency.

Also, do the tests really need 10 task pools?

Reply:
Fixed, and reduced to 4.